inventory/splunk_defaults_linux.yml

```diff
@@ -70,13 +77,19 @@ splunk:
         replication_factor: 3
         replication_port: 9887
     dfs:
+        dfs_home: "/opt/dfs"
+        spark_home: "/opt/spark"
+        java_home: "/opt/jdk"
         enable: False
-        port: 9000
+        port: 7777
         dfc_num_slots: 4
         dfw_num_slots: 10
         dfw_num_slots_enabled: False
         spark_master_host: "127.0.0.1"
-        spark_master_webui_port: 8080
```

Comment:
I believe changing these might break @mikedickey's work on https://github.com/splunk/splunk-operator/ and https://github.com/splunk/docker-spark with